controllers/suite_test.go

```diff
@@ -86,9 +87,6 @@ var _ = BeforeSuite(func(done Done) {
 	Expect(err).ToNot(HaveOccurred())
 	Expect(mgr).ToNot(BeNil())
 
-	err = NewAddonReconciler(mgr, ctrl.Log.WithName("controllers").WithName("Addon")).SetupWithManager(mgr)
```

Comment:
Why are these lines removed, we still want to test setup? We

Reply:
Because we moved new way to add controller with manager.